requirements/base.txt

```diff
@@ -1,129 +1,32 @@
 #
--e git+https://github.com/gridsingularity/d3a-interface@master#egg=d3a_interface
-    # via -r requirements/base.in
-appdirs==1.4.4
-    # via
-    #   d3a-interface
-    #   virtualenv
-attrs==20.3.0
-    # via
-    #   d3a-interface
-    #   jsonschema
-base58==2.0.1
-    # via
-    #   scalecodec
-    #   substrate-interface
-certifi==2020.12.5
-    # via
-    #   d3a-interface
-    #   requests
-    #   substrate-interface
-chardet==3.0.4
-    # via
-    #   d3a-interface
-    #   requests
-distlib==0.3.1
-    # via
-    #   d3a-interface
-    #   virtualenv
-filelock==3.0.12
-    # via
-    #   d3a-interface
-    #   tox
-    #   virtualenv
-idna==2.10
-    # via
-    #   d3a-interface
-    #   requests
-    #   substrate-interface
-jsonschema==3.2.0
-    # via d3a-interface
-kafka-python==2.0.2
-    # via d3a-interface
-more-itertools==8.8.0
-    # via scalecodec
-packaging==20.7
-    # via
-    #   d3a-interface
-    #   tox
-pendulum==2.1.2
-    # via d3a-interface
-pluggy==0.13.1
-    # via
-    #   d3a-interface
-    #   tox
-py==1.9.0
-    # via
-    #   d3a-interface
-    #   tox
-py-bip39-bindings==0.1.7
-    # via substrate-interface
-py-ed25519-bindings==0.1.3
-    # via substrate-interface
-py-sr25519-bindings==0.1.3
-    # via substrate-interface
-pyparsing==2.4.7
-    # via
-    #   d3a-interface
-    #   packaging
-pyrsistent==0.17.3
-    # via
-    #   d3a-interface
-    #   jsonschema
-python-dateutil==2.8.1
-    # via
-    #   d3a-interface
-    #   pendulum
-pytzdata==2020.1
-    # via
-    #   d3a-interface
-    #   pendulum
-redis==3.5.3
-    # via d3a-interface
-requests==2.25.1
-    # via
-    #   d3a-interface
-    #   scalecodec
-    #   substrate-interface
-scalecodec==0.11.16
-    # via substrate-interface
-six==1.15.0
-    # via
-    #   d3a-interface
-    #   jsonschema
-    #   python-dateutil
-    #   tox
-    #   virtualenv
-    #   websocket-client
-substrate-interface==0.13.9
-    # via -r requirements/base.in
-toml==0.10.2
-    # via
-    #   d3a-interface
-    #   tox
-tox==3.20.1
-    # via
-    #   -r requirements/base.in
-    #   d3a-interface
-urllib3==1.26.4
-    # via
-    #   d3a-interface
-    #   requests
-virtualenv==20.2.2
-    # via
-    #   d3a-interface
-    #   tox
-websocket-client==0.58.0
-    # via substrate-interface
-websockets==8.1
-    # via d3a-interface
-xxhash==2.0.2
-    # via substrate-interface
-
-# The following packages are considered to be unsafe in a requirements file:
-# setuptools
+-e git+https://github.com/gridsingularity/d3a-interface@update_dependencies#egg=d3a_interface  # via -r requirements/base.in
+attrs==21.2.0             # via d3a-interface, jsonschema
+base58==2.0.1             # via scalecodec, substrate-interface
+certifi==2021.10.8        # via d3a-interface, requests, substrate-interface
+chardet==4.0.0            # via d3a-interface, requests
+idna==2.10                # via d3a-interface, requests, substrate-interface
```

Comment:
You probably need to update this environment's pip tools, and recompile after reverting the d3a-interface branch

Reply:
tested with recreating environment and reinstallation of pip tools